packages/console/src/pages/ApplicationDetails/components/Settings.tsx

```diff
@@ -64,6 +64,16 @@ const Settings = ({ applicationType, oidcConfig, defaultData, isDeleted }: Props
           placeholder={t('application_details.description_placeholder')}
         />
       </FormField>
+      {applicationType === ApplicationType.Traditional && (
+        <FormField title="application_details.application_secret" className={styles.textField}>
+          <CopyToClipboard
+            isContentHidden
```

Comment:
Do we have a default value to determine whether contents in some CopyToClipboard are displayed by default?
I cannot find another appearance of `isContentHidden` in this file to determine its value.

Reply:
`isContentHidden` is enough, its default value is false.